merge.py

```diff
@@ -0,0 +1,188 @@
+"""Mergebot is a script which talks to GitHub and submits all ready pull requests.
+
+Mergebot talks to a specified GitHub project and watches for @mentions for its account.
+Acceptable commands are:
+  @<mergebot-name> merge
+"""
+from subprocess import call
+import glob
+import requests
+import sys
+import time
+
+AUTHORIZED_USERS = ["davor"]
+BOT_NAME = 'beam-testing'
+GITHUB_ORG = 'apache'
+REPOSITORY = 'incubator-beam'
+SECRET_FILE = '../../github_auth/apache-beam.secret'
+SOURCE_REMOTE = 'github'
+TARGET_BRANCH = 'master'
+TARGET_REMOTE = 'apache'
+
+GITHUB_API_ROOT = 'https://api.github.com'
+GITHUB_REPO_FMT_URL = GITHUB_API_ROOT + '/repos/{0}/{1}'
+GITHUB_REPO_URL = GITHUB_REPO_FMT_URL.format(GITHUB_ORG, REPOSITORY)
+CMDS = ['merge']
+ISSUES_URL = GITHUB_REPO_URL + '/issues'
+COMMENT_FMT_URL = ISSUES_URL + '/{pr_num}/comments'
+PULLS_URL = GITHUB_REPO_URL + '/pulls'
+
+bot_key = ''
+
+
+def main():
+  print('Starting up.')
+  # Load github key from filesystem
+  configs = {}
+  for filename in glob.iglob("config/*.json"):
+    with open(filename) as cfg:
+      configs[cfg['name']] = cfg
+  bot_key = ""
+  with open(SECRET_FILE, 'r') as key_file:
+    bot_key = key_file.read().strip()
+  print('Loaded key file.')
+  # Loop: Forever, once per minute.
+  while True:
+    poll_github()
+    time.sleep(60)
+
+def poll_github():
+
+
+def search_pr(pr):
+  pr_num = pr['number']
+  print('Looking at PR #{}.'.format(pr_num))
+  # Load comments for each pull request
+  cmt_url = COMMENT_FMT_URL.format(pr_num=pr_num)
+  print('Loading comments.')
+  r = requests.get(cmt_url, auth=(BOT_NAME, bot_key))
+  if r.status_code != 200:
+    print('Oops, that didn\'t work. Error below, moving on.')
+    print(r.text)
+    return
+
+  cmt_json = r.json()
+  if len(cmt_json) < 1:
+    print('No comments on PR #{}. Moving on.'.format(pr_num))
+    return
+  # FUTURE: Loop over comments to make sure PR has been LGTMed
+  cmt = cmt_json[-1]
+  cmt_body = cmt['body'].encode('ascii', 'ignore')
+  # Look for @apache-beam request comments
+  # FUTURE: Look for @apache-beam reply comments
+  if not cmt_body.startswith('@apache-beam'):
+    print('Last comment: {}, not a command. Moving on.'.format(cmt_body))
+    return
+  cmd_str = cmt_body.split('@apache-beam ', 1)[1]
+  cmd = cmd_str.split(' ')[0]
+  if cmd not in CMDS:
```

Comment:
If we have the list, I'd suggest printing the support commands. May also be worth having it be a map from command to the function to run, so that we don't need to update two places to add commands (both `CMDS` and the below if statement).